### PR TITLE
meta-scripts-improvements

### PR DESCRIPTION
**Pull request by mistyron**
updated meta-mistysom to develop @HEAD and updated meta files for creating and running the build container

### CODE REVIEW

File: Build/run.sh [hunk 1]
```diff
@@ -56,43 +55,36 @@ while [[ $# -gt 0 ]]; do
...
-		chmod -R 777 ${CPATH}/downloads 2>/dev/null
-		chmod -R 777 ${CPATH}/sstate-cache/${MPU} 2>/dev/null
-	else
-		chmod -R 777 ${CPATH}/downloads
-		chmod -R 777 ${CPATH}/sstate-cache/${MPU}
+		echo "Unable to obtain write permissions to ${CPATH} and its sub directories, edit the permissions of ${CPATH} accordingly! exit"
+		exit -1
 	fi
-	/usr/bin/docker run --privileged ${USE_TTY} --rm -v "${PWD}/${OUTDIR}":/home/yocto/rzg_vlp_v3.0.0/out -v "${CPATH}/downloads":/home/yocto/rzg_vlp_v3.0.0/build/downloads -v "${CPATH}/sstate-cache/${MPU}/":/home/yocto/rzg_vlp_v3.0.0/build/sstate-cache -e NO=${NO} -e SDK=${SDK} -e DLOAD=${DLOAD} --name ${CONTNAME} ${CONTNAME}
+	/usr/bin/docker run --privileged ${USE_TTY} --rm -v "${PWD}/${OUTDIR}":/home/yocto/rzg_vlp_v3.0.0/out -v "${CPATH}/downloads":/home/yocto/rzg_vlp_v3.0.0/build/downloads -v "${CPATH}/sstate-cache":/home/yocto/rzg_vlp_v3.0.0/build/sstate-cache -e NO=${NO} -e SDK=${SDK} -e DLOAD=${DLOAD} --name ${CONTNAME} ${CONTNAME}
```
Comment (matinlotfali) [inline Build/run.sh:89]: Is there a reason that you removed the `${MPU}` from the cache directory?

File: Build/start.sh [hunk 2]
```diff
@@ -28,9 +27,10 @@ echo "    CONFIGURATION COPIED TO conf/"
 if [ -z $DLOAD ];
 then
 	cd $WORK/build
-	7z x ~/oss_pkg_v3.0.0.7z
+	7z x ~/oss_pkg_rzv_v3.0.0.7z
```
Comment (matinlotfali) [inline Build/start.sh:30]: Is there a reason that you are using the `rzv` instead?

File: Build/start.sh [hunk 3]
```diff
@@ -28,9 +27,10 @@ echo "    CONFIGURATION COPIED TO conf/"
...
 fi
 ##Apply DRPAI patch
+echo "IMAGE_INSTALL_append = \" gstreamer1.0-drpai ai-eva-sw\"" >> ${WORK}/meta-mistysom/recipes-core/images/mistysom-image.bbappend
```
Comment (matinlotfali) [inline Build/start.sh:33]: This should not be added to the G2L repo. It is specific to V2L

File: Build/start.sh [hunk 4]
```diff
@@ -53,14 +53,14 @@ fi
...
 
 echo "    ------------------------------------------------
     SETUP SCRIPT BUILD ENVIRONMENT SETUP SUCCESSFUL!
     run the following commands to start the build:
-    'cd ~/rzg_vlp_v3.0.0/'
+    'cd ${WORK}'
     'source poky/oe-init-build-env'
     'bitbake mistysom-image'"
-cd ~/rzg_vlp_v3.0.0
+cd ~/rzv_vlp_v3.0.0
```
Comment (matinlotfali) [inline Build/start.sh:65]: This should not change.

**Review by matinlotfali — CHANGES_REQUESTED** [review]

File: Build/run.sh (hunk 1 above)
Comment (mistyron) [reply Build/run.sh:89]: WIP

File: Build/start.sh (hunk 2 above)
Comment (mistyron) [reply Build/start.sh:30]: WIP

File: Build/start.sh (hunk 3 above)
Comment (mistyron) [reply Build/start.sh:33]: Got it! Of course!

File: Build/start.sh (hunk 4 above)
Comment (mistyron) [reply Build/start.sh:65]: WIP

**Review by matinlotfali — APPROVED** [review]

**Pull request merged**